samples/python/assign-reviewers.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python
+
+###############################################################################
+''' Assigns reviewers to papers
+ ex. python assign-reviewers.py --baseurl http://localhost:3000
+       --username admin --password admin_pw 'reviewer@gmail.com,3'
+
+ Checks paper number valid.
+ Check reviewer email address or domain is not on the conflicts list.
+ Check reviewer is in the system.
+ If reviewer is not in conference reviewers group (config.CONF/Reviewers), add it.
+ If reviewer not already assigned to this paper:
+	Determine AnonReviewer number
+	Create Paper#/AnonReviewer#  group with this reviewer as a member
+    Assign Paper#/AnonReviewer# to the Paper#/Reviewers group and
+                                Paper#/Reviewers/NonReaders group for this paper'''
+###############################################################################
+
+## Import statements
+import argparse
+import csv
+import sys
+import config
+from openreview import *
+
+## Argument handling
+parser = argparse.ArgumentParser()
+parser.add_argument('assignments', help="either (1) a csv file containing reviewer assignments or (2) a string of the format '<email_address>,<paper#>' e.g. 'reviewer@cs.umass.edu,23'")
+parser.add_argument('--baseurl', help="base url")
+parser.add_argument('--username')
+parser.add_argument('--password')
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+if args.username!=None and args.password!=None:
+    client = Client(baseurl=args.baseurl, username=args.username, password=args.password)
```

Comment:
you should be able to remove the "if/else" block here, and just have a single line like this one